source/client/flush_worker_impl.h

```diff
@@ -0,0 +1,51 @@
+// Flush worker implementation. Flush worker periodically flushes metrics
+// snapshot to all configured stats sinks in Nighthawk.
+#pragma once
+
+#include <vector>
+
+#include "envoy/api/api.h"
+#include "envoy/event/dispatcher.h"
+#include "envoy/stats/sink.h"
+#include "envoy/stats/store.h"
+#include "envoy/thread_local/thread_local.h"
+
+#include "common/worker_impl.h"
+
+namespace Nighthawk {
+namespace Client {
+
+// Only a single live flush worker instance can be created in Nighthawk at any
+// time.
+// Flush worker periodically flushes metrics snapshot to all configured stats sinks in Nighthawk. It
+// will keep running until exitDispatcher() gets called after all client workers are completed in
+// process_impl.cc. It will make the last flush before shutdown in shutdownThread().
+class FlushWorkerImpl : public WorkerImpl {
+public:
+  FlushWorkerImpl(Envoy::Api::Api& api, Envoy::ThreadLocal::Instance& tls,
+                  Envoy::Stats::Store& store,
+                  std::list<std::unique_ptr<Envoy::Stats::Sink>>& stats_sinks,
+                  const std::chrono::milliseconds& stats_flush_interval);
```

Comment:
Per the style guide the input-only arguments (e.g. const references) should come before input/output and output only arguments.

https://google.github.io/styleguide/cppguide.html#Inputs_and_Outputs

Reply:
Thanks Jakub. Done